### DISCUSSION TITLE
What is the easiest way to create roles and apply them?

### DISCUSSION THREAD

**Discussion by JamesSingleton** [supabase/supabase#26731] [Questions]
Is there a way to add roles in a _easy_ way? I know easy is relative but maybe via the UI or something? I was going back and forth between Clerk and Supabase Auth for my auth and thought maybe I keep everything together with Supabase Auth. Clerk has a way where you can add metadata like a "role" to a user via their UI. However, it doesn't look like you can do the same with Supabase? It looks like I have to create a `.sql` file under migrations? Is there something I am missing? I'm just trying to create an admin role and a voter role for my application.

**Comment by GaryAustin1** [comment]
A couple of links.  But it is roll your own.

https://supabase.com/docs/guides/database/postgres/custom-claims-and-role-based-access-control-rbac
https://github.com/orgs/supabase/discussions/11948

**Reply by JamesSingleton** [reply]
Ah ok, I’ve gone through those but I was hoping there was a easier way through the UI or something 🫤
